.github/CODEOWNERS

```diff
@@ -1,49 +1,63 @@
 # https://help.github.com/articles/about-codeowners/
 
-/distribution/ @a-rodin
+PRIVACY.md @binarylogic @jamtur01
+LICENSE @binarylogic @jamtur01
+CONTRIBUTING.md @binarylogic @jamtur01
+CODE_OF_CONDUCT.md @binarylogic @jamtur01
+NOTICE @binarylogic @jamtur01
+SECURITY.md @binarylogic @jamtur01
+
+.github/workflows./* @jamtur01
+.github/semantic* @binarylogic @jamtur01
+.github/SUPPORT.md @binarylogic @jamtur01
+.github/PULL_REQUEST_TEMPLATE.md @binarylogic @jamtur01
+.github/ISSUE_TEMPLATE/* @binarylogic @jamtur01
+
+/distribution/ @hovebear @jamtur01
 /distribution/kubernetes/ @MOZGIII
 /distribution/helm/ @MOZGIII
 
 /proto/ @lukesteensen
 
-/scripts/build-* @a-rodin
-/scripts/ci-docker-images/ @a-rodin
-/scripts/check-* @a-rodin
-/scripts/docker-* @a-rodin
+/scripts/build-* @hovebear
+/scripts/ci-docker-images/ @hovebear @jamtur01 
+/scripts/check-* @jamtur01
+/scripts/docker-* @jamtur01
+/scripts/environment/* @hovebear @jamtur01
 /scripts/generate* @binarylogic
-/scripts/package-* @a-rodin
-/scripts/release-* @a-rodin
-/scripts/upgrade-* @a-rodin
+/scripts/package-* @hovebear
+/scripts/release-* @hovebear
+/scripts/upgrade-* @hovebear
 /scripts/ci-setup-minikube.sh @MOZGIII
 /scripts/ci-setup-helm.sh @MOZGIII
 /scripts/release-helm.sh @MOZGIII
 /scripts/deploy-kubernetes-test.sh @MOZGIII
 /scripts/test-e2e-kubernetes.sh @MOZGIII
 
-/src/buffers/ @a-rodin
+/src/buffers/ 
```

Comment:
Do we need to list sources that have no owner?

Reply:
I left them in so people could see them and claim them. I'll remove any we decide not to claim.